fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -52,6 +52,7 @@ import org.mozilla.fenix.theme.FirefoxTheme
  * @param onProductRecommendationsEnabledStateChange Invoked when the user changes the product
  * recommendations toggle state.
  * @param onReviewGradeLearnMoreClick Invoked when the user clicks to learn more about review grades.
+ * @param onBylineLinkClick Invoked when the user clicks on the byline link.
```

Comment:
@MozillaNoah I think this was lost in communication, let's also update all byline references to footer here.

Reply:
Ah, dang it. I just missed it. I'll make a ticket